api/sns.ts

```diff
@@ -1,8 +1,37 @@
 import { api } from 'api/instance';
 
+import { ReactionType } from 'typings/reviews';
+
 const SNS_URL = '/sns';
 
 export const snsAPI = {
+  getFeed: async () => {
+    const SIZE = 10;
+
+    return await api
+      .get(`${SNS_URL}/feeds`, {
+        params: {
+          size: SIZE,
+        },
+      })
+      .then((res) => res.data);
+  },
+  addReaction: async (reviewId: number, reaction: ReactionType) => {
+    const params = { reaction };
+
+    return await api.post(`${SNS_URL}/review-reaction/${reviewId}`, null, {
+      params,
+    });
+  },
+  deleteReaction: async (reviewId: number) => {
+    return await api.delete(`${SNS_URL}/review-reaction/${reviewId}`);
+  },
+  addScrap: async (reviewId: number) => {
+    return await api.post(`${SNS_URL}/scrap-reviews/${reviewId}`);
+  },
+  deleteScrap: async (reviewId: number) => {
+    return await api.delete(`${SNS_URL}/scrap-reviews/${reviewId}`);
+  },
```

Comment:
```suggestion
  deleteReaction: (reviewId: number) => {
    return api.delete(`${SNS_URL}/review-reaction/${reviewId}`);
  },
  addScrap: (reviewId: number) => {
    return api.post(`${SNS_URL}/scrap-reviews/${reviewId}`);
  },
  deleteScrap: (reviewId: number) => {
    return api.delete(`${SNS_URL}/scrap-reviews/${reviewId}`);
  },
```

단순히 response를 return 해주는 경우에는 async await를 사용하지 않아도 괜찮을 것 같아요!

Reply:
단순히 response를 return 해주는 경우: 가 무슨 뜻일까요..? axios가 promise api를 활용하는 비동기 라이브러리이기 때문에 비동기 코드가 중복된다는 말씀이실까요??

Reply:
여기서 async await를 제거해줘도 동일한 결과값이 나오기 때문에 제거해줘도 괜찮아보인다. 라는걸 말하고 싶었습니다 ㅎㅎ